packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -69,61 +69,54 @@ const Basis = ({
     };
 
     return (
-        <div className='trade-params__strike'>
-            {contract_type === 'vanilla' && (
```

Comment:
MinMaxStakeInfo generalized component will be used for both Turbos & Vanillas, as discussed with designers.